libs/cardano-ledger-core/testlib/Test/Cardano/Ledger/Core/Binary/RoundTrip.hs

```diff
@@ -216,6 +216,11 @@ roundTripCoreEraTypesSpec = do
     roundTripAnnEraSpec @era @(TxWits era)
     roundTripAnnEraSpec @era @(TxBody era)
     roundTripAnnEraSpec @era @(Tx era)
+    prop ("MemPack/CBOR Roundtrip " <> show (typeRep $ Proxy @(TxOut era))) $
```

Comment:
Cool that this roundtrip mechanism is versatile, nice to see it reused like this.